src/web/docusaurus/docs/getting-started/terraform.md

```diff
@@ -0,0 +1,109 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Setup resources with Terraform
```

Comment:
I think this needs a note about getting an AWS account in the Learner Lab.  It also needs some warning about costs, since this will cost a lot if you start an r5.large and leave it.

Reply:
I agree, we can leave a note saying that there are two ways to stop the EC2 instance: 

using the aws CLI in learner lab ` aws ec2 stop-instances --instance-ids {instance-id}`
 
or manually go into the EC2 service in AWS learner lab and select the ec2 instance, choose instance dropdown and stop instance.

I'm wondering if this can go under destroy your resources?